phpcs.xml

```diff
@@ -15,23 +15,39 @@
     <exclude-pattern>*/vendor/*</exclude-pattern>
     <exclude-pattern>*/libs/*</exclude-pattern>
 
+    <!-- exclude all plugins included as submodule as long as they might not fully follow this CS -->
+    <exclude-pattern>plugins/(AnonymousPiwikUsageMeasurement|Bandwidth|CustomAlerts|CustomVariables|DeviceDetectorCache|LogViewer|LoginLdap|MarketingCampaignsReporting|Provider|QueuedTracking|SecurityInfo|TagManager|TrackingSpamPrevention|TreemapVisualization|VisitorGenerator)/*</exclude-pattern>
+
     <config name="installed_paths" value="../../slevomat/coding-standard"/>
 
-    <!-- Forbid other line endings than \n -->
-    <rule ref="Generic.Files.LineEndings">
-        <properties>
-            <property name="eolChar" value="\n" />
-        </properties>
+    <!-- Use PSR standard as default -->
+    <!-- Some rules are currently excluded/restricted, but those restrictions should be removed in the future -->
+
+    <rule ref="PSR1">
+        <!-- can't use this as long as we have functions.php files that need to be required manually -->
+        <exclude name="PSR1.Files.SideEffects.FoundWithSymbols" />
     </rule>
 
-    <!-- Expect a file to end with a new line -->
-    <rule ref="PSR2.Files.EndFileNewline"/>
```

Comment:
Keep this PSR2 rule

Reply:
ah, didn't want to remove that one, till we have PSR2 or PSR12 as base